docs/features.md

```diff
@@ -0,0 +1,43 @@
+---
+Order: 20
+Title: Features
+Description: Features of the Cake.Issues.DocFx addin.
+---
+The [Cake.Issues.DocFx addin] provides the following features.
+
+# Basic features
+
+* Reads warnings from [DocFx] log files.
+
+  :::{.alert .alert-info}
+  [DocFx] can be run with [Cake.DocFx] addin.
+  :::
+
+# Supported comment formats
+
+|                                                                    | Comment format                 | Remarks                        |
+|--------------------------------------------------------------------|--------------------------------|--------------------------------|
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IssueCommentFormat.PlainText` |                                |
+| <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IssueCommentFormat.Markdown`  |                                |
+| <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IssueCommentFormat.Html`      |                                |
+
+# Supported IIssue properties
+
+|                                                                    | Property                          | Remarks                        |
+|--------------------------------------------------------------------|-----------------------------------|--------------------------------|
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.ProviderType`             |                                |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.ProviderName`             |                                |
+| <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IIssue.ProjectName`              |                                |
+| <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IIssue.ProjectFileRelativePath`  |                                |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.AffectedFileRelativePath` |                                |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.Line`                     |                                |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.Message`                  |                                |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.Priority`                 | Always [IssuePriority.Warning] |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.PriorityName`             | Always [IssuePriority.Warning] |
```

Comment:
`Always [IssuePriority.Warning]` seems not correct for `IIssue.PriorityName`.

Reply:
You're right in the meantime there could also be suggestions